weed/shell/command_volume_resume_vacuum.go

```diff
@@ -0,0 +1,41 @@
+package shell
+
+import (
+	"context"
+	"io"
+
+	"github.com/seaweedfs/seaweedfs/weed/pb/master_pb"
+)
+
+func init() {
+	Commands = append(Commands, &commandResumeVacuum{})
+}
+
+type commandResumeVacuum struct {
+}
+
+func (c *commandResumeVacuum) Name() string {
+	return "volume.resumevacuum"
```

Comment:
how about using `volume.vaccum.resume`? And also change the file names accordingly.

Reply:
Done!